internal/renderers/skia/Cargo.toml

```diff
@@ -49,7 +49,7 @@ glutin = { version = "0.30", default-features = false, features = ["egl", "wgl"]
 winit = { version = "0.28.1", optional = true, default-features = false }
 
 [target.'cfg(target_family = "windows")'.dependencies]
-winapi = { version = "0.3", features = ["dwrite", "d3d12", "dxgi", "dxgi1_2", "dxgi1_3", "dxgi1_4", "d3d12sdklayers", "synchapi", "winbase"] }
+winapi = { version = "0.3", features = ["impl-default", "dwrite", "d3d12", "dxgi", "dxgi1_2", "dxgi1_3", "dxgi1_4", "d3d12sdklayers", "synchapi", "winbase"] }
```

Comment:
Is that related?

Reply:
Yes. Previously this feature was implicitly selected (through dark-light-mode). Without that dependency we need to select it explicitly.